cime_config/machines/cmake_macros/intel_alvarez.cmake

```diff
@@ -15,24 +15,18 @@ set(SFC "ifx")
 # on the compiler ID or version, so for now, simply manually adjust the CXXFLAG setting for pm-cpu/intel
 # Try to manually remove -fp-model=source (and replace with -fp-model=precise) from CXXFLAGS
 #message(STATUS "ndk CXXFLAGS=${CXXFLAGS}")
-set(CXXFLAGS " ") # hardcode it here to blank, then try to do same things as in intel.cmake
+set(CMAKE_CXX_FLAGS " ") # hardcode it here to blank, then try to do same things as in intel.cmake
```

Comment:
Again, this PR is simply converting stuff, but what's the reason of not using what's in the generic `intel.cmake`?